src/bosh_aws_cpi/lib/cloud/aws/aws_provider.rb

```diff
@@ -2,36 +2,26 @@ module Bosh::AwsCloud
   class AwsProvider
     include Helpers
 
-    attr_reader :ec2_client, :ec2_resource, :alb_client, :elb_client
+    attr_reader :ec2_client, :ec2_resource, :alb_client, :elb_client, :elb_params, :ec2_params
```

Comment:
The idea of the `AwsProvider` is to encapsulate and hide detailed logic of parameters, and expose only what is necessary.
Also, it's not a recommended practice to make methods or attributes public only for testing.

Reply:
Not sure how else to test this at a unit level.  Since no way to validate parameter maps are getting built correctly without being able to assert values within them.  And majority of logic in this provider is marshalling values into these params.  Open to suggestions on how to better test this. @cdutra